tensorflow/python/keras/optimizer_v2/adadelta_test.py

```diff
@@ -75,16 +75,16 @@ def doTestBasic(self, use_resource=False, use_callable_params=False):
             slot = [None] * 2
             slot_update = [None] * 2
             slot[0] = adadelta_opt.get_slot(var0, "accum_grad")
-            self.assertEqual(slot[0].get_shape(), var0.get_shape())
+            self.assertEqual(slot[0].shape, var0.get_shape())
```

Comment:
Why not here as well?

Reply:
@omalleyt12 , thanks for the review, i have updated the code as per your suggestion. Kindly check and approve.

Regards
Amit